docs/testplan/BGP-T2-Anchor-test-plan.md

```diff
@@ -0,0 +1,38 @@
+# BGP T2 Anchor test plan
```

Comment:
Can you change Anchor to "Anchor prefixes"

Reply:
Addressed